tests/execution_engine/test_sqlalchemy_execution_engine.py

```diff
@@ -790,6 +791,74 @@ def test_resolve_metric_bundle_with_nonexistent_metric(sa):
         print(e)
 
 
+def test_resolve_metric_bundle_with_compute_domain_kwargs_json_serialization(sa):
+    """
+    Insures that even when "compute_domain_kwargs" has multiple keys, it will be JSON-serialized for "IDDict.to_id()".
+    """
```

Comment:
you even did an `sa` one! :) thank you

Reply:
@Shinnnyshinshin It had to be both -- they are subject to the same situation (`RowCondition` which begins as not yet serialized).  Thanks!